app/views/bookings/index.html.erb

```diff
@@ -0,0 +1,11 @@
+<div class="container">
+<div class="row">
```

Comment:
This needs to be indented inside the container... and h1 and below indented inside the row